### ISSUE TITLE
Need a “edit selection” button in mobile view

### ISSUE THREAD

**Issue by CeciliaCY** [NYCPlanning/labs-nyc-factfinder#252]
It’s hard to know how to edit selection on Mobile View.

**Comment by CeciliaCY** [comment]
![127086ED-5F91-480C-8E0B-E81B5C31A58B.jpeg](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59ee0299cfdf5100126377c2/125516c66e82c728ace21e0d46b9c6cd73dad58caba4ea65fd0cbdc07116043e7121f3093da0bb799d1758525e2e4ba4334a4356bcbbd671f3e57862804e0ee484650b76c26c13a4b0d64bbb01ba26df4a81b2bc8e45d2f42f67b6d5db1063474f4dcbed5727a19e41c95df749e34e54a51a346ae2742431.jpeg)

**Comment by andycochran** [comment]
Good catch, @CeciliaCY. I can handle this. Now, by default the text is hidden, showing on hover. I can make it visible on small screens and hide/show on larger screens — which are more likely to not be touch devices.
